pkg/controller/operands/operand.go

```diff
@@ -112,6 +112,8 @@ func (h *genericOperand) handleExistingCr(req *common.HcoRequest, key client.Obj
 	}
 
 	if updated {
+		// update resourceVersions of objects in relatedObjects
```

Comment:
https://github.com/openshift/custom-resource-status/blob/master/objectreferences/README.md is not that clear on this,
but I tend to think that this is enough.